torch/_torch_docs.py

```diff
@@ -6699,11 +6699,13 @@ def merge_dicts(*dicts):
 
 Computes the matrix-matrix multiplication of a product of Householder matrices with a general matrix.
 
-Multiplies a matrix `C` (given by :attr:`other`) with a matrix `Q`,
+Multiplies a :math:`m \times n` matrix `C` (given by :attr:`other`) with a matrix `Q`,
 where `Q` is represented using Householder reflectors `(input, tau)`.
 See `Representation of Orthogonal or Unitary Matrices`_ for further details.
 
 If :attr:`left` is `True` then `op(Q)` times `C` is computed, otherwise the result is `C` times `op(Q)`.
+When ``left==True`` the implicit matrix `Q` has the size  :math:`m \times m`,
```

Comment:
Perhaps a better wording:

> When :attr:`left` is `True`, the implicit matrix `Q` has size :math:`m \times m`. It has size :math:`n \times n` otherwise.